scalafix-tests/unit/src/main/scala/scalafix/test/cli/Rules.scala

```diff
@@ -0,0 +1,44 @@
+package scalafix.tests.cli
```

Comment:
the rules in this file are referenced in https://github.com/scalacenter/scalafix/blob/660877e92000c20b266e1e754cd01089d01c598d/scalafix-tests/unit/src/main/resources/META-INF/services/scalafix.v1.Rule#L8-L13 so they cannot be in `test/scala-2`